test/DelegationTokenInternalTest.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {Test, console} from "../lib/forge-std/src/Test.sol";
+import {DelegationToken} from "../src/DelegationToken.sol";
+
+contract DelegationTokenInternalTest is Test, DelegationToken {
+    uint256 internal constant MAX_TEST_AMOUNT = 1e28;
+
+    function getInitializableStorage() internal pure returns (InitializableStorage storage $) {
+        assembly {
+            $.slot := 0xf0c57e16840df040f15088dc2f81fe391c3923bec73e23a9662efc9c229c6a00
+        }
+    }
+
+    function _delegatedVotingPower(address account) internal view returns (uint256) {
+        DelegationTokenStorage storage $ = _getDelegationTokenStorage();
+        return $._delegatedVotingPower[account];
+    }
+
+    function testDisabledInitializers() public view {
+        InitializableStorage storage $ = getInitializableStorage();
+        assertEq($._initialized, type(uint64).max, "Initializers not disabled");
+    }
+
+    function testDelegationTokenStorageLocation() public pure {
+        bytes32 expectedSlot =
+            keccak256(abi.encode(uint256(keccak256("DelegationToken")) - 1)) & ~bytes32(uint256(0xff));
+        bytes32 usedSlot = DelegationTokenStorageLocation;
+        assertEq(expectedSlot, usedSlot, "Wrong slot used");
+    }
+
+    function testMoveDelegateVotesDifferentAccounts(
+        address from,
+        address to,
+        uint256 initialVoteFrom,
+        uint256 initialVoteTo,
+        uint256 amount
+    ) public {
+        // Setup
+        DelegationTokenStorage storage $ = _getDelegationTokenStorage();
+        initialVoteFrom = bound(initialVoteFrom, 0, MAX_TEST_AMOUNT);
+        $._delegatedVotingPower[from] = initialVoteFrom;
+        initialVoteTo = bound(initialVoteTo, 0, MAX_TEST_AMOUNT);
+        $._delegatedVotingPower[to] = initialVoteTo;
+
+        assertEq(_delegatedVotingPower(from), initialVoteFrom);
+        assertEq(_delegatedVotingPower(to), initialVoteTo);
+
+        // Test
+        amount = bound(amount, 0, initialVoteFrom);
+        _moveDelegateVotes(from, to, amount);
+
+        uint256 expectedVoteFrom = from == address(0) ? initialVoteFrom : initialVoteFrom - amount;
```

Comment:
I'd rather use the expanded syntax of conditional expressions.

Reply:
It's 6 lines of code instead of 1 though
```solidity
        uint256 expectedVoteFrom;
        if (from == address(0)) {
            expectedVoteFrom = initialVoteFrom;
         else {
            expectedVoteFrom = initialVoteFrom - amount;
        }
```
do you find it clearer ?

Reply:
To be honest, yes. But that's only me. The reason why I dislike oneliners is because they may escape the eye easily during review while they are often non trivial. Multiple lines force us to read step-by-step. In general, I tend to be very skeptical on syntactic sugar that compresses to very small code. I much prefer very simple code over several lines instead of short but super dense (semantically speaking) code, makes me think in some way to Go To Considered Harmful by E. Dijsktra.

Reply:
Consider this comment as optional.